client/styles/components/_modal.scss

```diff
@@ -1,7 +1,7 @@
 .modal {
   position: absolute;
   top: #{60 / $base-font-size}rem;
-  right: #{400 / $base-font-size}rem;
+  right: calc(50vw - #{(400 / $base-font-size) / 2}rem);
```

Comment:
my preference for this is
```scss
right: 50%;
transform: translate(50%, 0); // this might need to be translate(-50%, 0)? whatever centers it
```
as it's not dependent on the width of the modal!

Reply:
That is a better way of aligning the dialog box. Can I use ```translateX``` instead ?